### ISSUE TITLE
Bug: EventController->populate does not regenerate MISPObjectAttribute uuids

### ISSUE THREAD

**Issue by github-germ** [MISP/MISP#9012]
### Expected behavior

When using `Populate from... with MISPEvent JSON` that may have `MISPObjectAttribute.uuid` that already exist in the `attributes` table, and the user checks `Regenerate UUIDs` at the bottom of the pasted in textbox, the expectation is that all entity uuids within the JSON will be regenerated.

### Actual behavior

The populate does the import; however, any `MISPObjectAttribute.uuid` that already existing in `attributes.uuid` causes the parent `MISPObject` to be imported with none of the member `MISPObjectAttributes` in the JSON

### Steps to reproduce

Create a `MISPEvent` JSON that includes a `MISPObject` that has a `MISPObjectAttribute.uuid` that is already in `attributes.uuid`. Populate that JSON. Observe the object is empty.

Our test case had a >1M JSON with lots of objects. To further test, I replaced the first 8 hex-chars of every contained uuid with `ffffffff`, and that imported with all objects correctly populated.

This may be an area to review: in `EventsController.php` I may be wrong, but I don't think this traverses the `MISPObjectAttributes`:
```
    public function populate($id, $regenerateUUIDs=false)
    {
        ...
            $capturedObjects = ['Attribute', 'Object', 'Tag', 'Galaxy', 'EventReport'];
            foreach ($capturedObjects as $objectType) {
                if (!empty($this->request->data[$objectType])) {
                    if (!empty($regenerateUUIDs)) {
                        foreach ($this->request->data[$objectType] as $i => $ob$
                            unset($this->request->data[$objectType][$i]['id']);
                            unset($this->request->data[$objectType][$i]['uuid']$
                        }
                    }
                    $eventToSave['Event'][$objectType] = $this->request->data[$$
                }
            }



### Version

2.4.169

### Operating System

RH

### Operating System version

8.7

### PHP version

7.4.33

### Browser

_No response_

### Browser version

_No response_

### Relevant log output

_No response_

### Extra attachments

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by github-germ** [comment]
@righel I may have pinpointed the "fix" above. Is it possible to get this into 2.4.170 also? Thank you!

**Comment by righel** [comment]
Good catch, fixed, will be included in `v2.4.170`

**Comment by github-germ** [comment]
Thanks. Works with 2.4.170 installed here.
